### ISSUE TITLE
Fails on Multiple Inlines

### ISSUE THREAD

**Issue by mattdesl** [substack/brfs#35]
A single inline works fine, but multiple inlines do not:  

```js
var fs = require('fs').readFileSync(__dirname+'/index.js')
var fs2 = require('fs').readFileSync(__dirname+'/package.json')
console.log(fs)
```

**Comment by davidpfahler** [comment]
Same problem. The second I add more than one readFileSync calls, the build fails.

`Error: Line 18: Unexpected string while parsing file: ...`

**Comment by clayzermk1** [comment]
:+1: seeing this too

Moving `var fs = require('fs');` to its own line and using `fs` in place of the inline require calls seems to work.

**Comment by abarreir** [comment]
@clayzermk1 Worked for me, thanks!

**Comment by substack** [comment]
This was fixed upstream in static-module 1.1.2.

**Comment by cloakedninjas** [comment]
Seems to still be an issue?

```
npm ls static-module
project@ /Users/cloakedninjas/project
└─┬ brfs@1.4.3
  └── static-module@1.3.1
```

@clayzermk1 workaround fixes it though
